bin/generate-tiles

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+set -o errexit
+set -o pipefail
+set -o nounset
+
+# For backward compatibility, allow both PG* and POSTGRES_* forms,
+# with the non-standard POSTGRES_* form taking precedence.
+# An error will be raised if neither form is given, except for the PGPORT
+export PGHOST="${POSTGRES_HOST:-${PGHOST?}}"
```

Comment:
the way it is written now, you require that both `POSTGRES_HOST/PGHOST` and `PGHOST_LIST` be set if the user wants to use multiple hosts.  I think you should move the above line to the `else` block below -- i.e. when `PGHOST_LIST` is not set.

Reply:
Oh, yes. You are right. I didn't realize the user might have set just `PGHOSTS_LIST`